### PR TITLE
Settings and docs

### PR DESCRIPTION
**Pull request by lmccartney**
The changes to README.rst are a decent guide to the changes I made.

### CODE REVIEW

File: factoriomaps/settings/base.py [hunk 1]
```diff
@@ -66,7 +62,7 @@
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [os.path.join(BASE_DIR, 'templates')]
+        'DIRS': [os.path.join(BASE_DIR, '../../templates'), ]
```
Comment (Fudster) [inline factoriomaps/settings/base.py]: Is this change really needed?

Comment (lmccartney) [reply factoriomaps/settings/base.py]:
```suggestion
        'DIRS': [os.path.join(BASE_DIR, 'templates'), ]
```

Comment (lmccartney) [reply factoriomaps/settings/base.py]: just checked using shell, you're right
